### ISSUE TITLE
save subdspaths  does nothing

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#1389]
```shell
(venv-tests)3 10596.....................................:Sun 19 Mar 2017 11:17:47 AM EDT:.
(git)smaug:/mnt/datasets/datalad/crawl/openfmri[master]git
$> git diff ds000017[AB]
diff --git a/ds000017A b/ds000017A
index caec2d1..c69868f 160000
--- a/ds000017A
+++ b/ds000017A
@@ -1 +1 @@
-Subproject commit caec2d1df5e8ddabcee7916871880f8bf3889b0f
+Subproject commit c69868f434390bbde4422344e18a311bff85e924
diff --git a/ds000017B b/ds000017B
index 64f9195..76551c8 160000
--- a/ds000017B
+++ b/ds000017B
@@ -1 +1 @@
-Subproject commit 64f9195de0005f5aa511988fa19aabdbc37c8979
+Subproject commit 76551c816fbd13c0a9277f60a1791620de67c632
changes on filesystem:                                                                                                           
 ds000017A | 2 +-
 ds000017B | 2 +-
(venv-tests)3 10597.....................................:Sun 19 Mar 2017 11:17:51 AM EDT:.
(git)smaug:/mnt/datasets/datalad/crawl/openfmri[master]git
$> datalad save -m "updated A and B datasets for 17 for their final version - superseeeded by pure 017" ds000017[AB]
changes on filesystem:                                                                                                           
 ds000017A | 2 +-
 ds000017B | 2 +-
```

so those two sub-datasets progressed forward, but if I specify them to save command -- it just does nothing.  Have to use `-u` option to save their adjusted state

**Comment by yarikoptic** [comment]
can anyone of save experts look at it @datalad/developers ?  keep hitting it

**Comment by mih** [comment]
Fix in #1506
